OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -43,12 +43,14 @@
       <Private>False</Private>
     </Reference>
     <Reference Include="System" />
+    <Reference Include="System.Drawing" />
```

Comment:
```return type.GetInterfaces().Contains(typeof(ISync))``` produces these errors (until you include the dll) - don't ask me!

A loop seems to avoid the problem - I'll use that.